main.go

```diff
@@ -31,7 +31,7 @@ import (
 )
 
 var (
-	VERSION = "vlatest"
+	VERSION = "0.6.0"
```

Comment:
Please undo this change. We inject the version number in the build process.

Reply:
@kylegc - reverted, thanks; didn't know about this.